openpype/pipeline/workfile/lock_workfile.py

```diff
@@ -0,0 +1,67 @@
+import os
+import json
+from uuid import uuid4
+from openpype.lib.pype_info import get_workstation_info
+
+
+def _read_lock_file(lock_filepath):
+    with open(lock_filepath, "r") as stream:
+        data = json.load(stream)
+    return data
+
+
+def _get_lock_file(filepath):
+    return filepath + ".lock"
```

Comment:
What happens if potentially an application's workfile does have a `.lock` file of its own. Should we maybe add a safeguard in `_read_lock_file` with some error logging if the file wasn't readable or laid out as OP expected?

Reply:
Good point. My suggestion is `.oplock` but any other suggestions are welcomed :)